whois-api/src/test/java/net/ripe/db/whois/api/ElasticSearchHelper.java

```diff
@@ -0,0 +1,70 @@
+package net.ripe.db.whois.api;
+
+import net.ripe.db.whois.common.dao.jdbc.DatabaseHelper;
+import org.apache.http.HttpHost;
+import org.elasticsearch.action.admin.indices.delete.DeleteIndexRequest;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.client.indices.CreateIndexRequest;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ElasticSearchHelper {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseHelper.class);
+
+    public static final String ELASTIC_HOSTNAME = System.getProperty("elastic.host");
```

Comment:
Can you make this class a Component and inject properties?

Reply:
Yes, initial idea was having it as a util class but it makes more sense to make a component now